cmd/galaxy_install_test.go

```diff
@@ -89,5 +119,9 @@ func TestGalaxyInstallRun(t *testing.T) {
 		if !strings.Contains(combined, tc.out) {
 			t.Errorf("expected output %q to contain %q", combined, tc.out)
 		}
+
+		for _, file := range tc.roleFiles {
+			os.Remove(file)
```

Comment:
How about move this to immediately after line 108 and use `defer`?

Reply:
I had that originally but `defer` only runs before exit and this needs to be removed on *each* loop iteration.

Reply:
My bad